### DISCUSSION TITLE
Installing on Nano Pi R4S without a console

### DISCUSSION THREAD

**Discussion by kfoxb** [siderolabs/talos#6812] [General]
I'm considering buying some Nano Pi's to run Talos on and I was looking over the documentation [here](https://www.talos.dev/v1.3/talos-guides/install/single-board-computers/nanopi_r4s/) and it says:
>Insert the SD card to your board, turn it on and wait for the console to show you the instructions for bootstrapping the node.

There's no console output on the Nano Pi though, it's headless. Is there a way around that if I know the IP address assigned to the machine? Or should I look into some kind of UART display to connect to the board for this?

**Comment by rsmitty** [comment]
Hey @kfoxb, hope you're well.  Assuming the rpi image "just works" for the nano, you can probably cheese this if you know the IP beforehand. You'd basically boot talos, then check that apid is up and talos is in maintenance mode by running one of the supported insecure commands ( `talosctl --endpoint <known-ip> --nodes <known-ip>  disks --insecure` for example). If that returns as expected, you can simply apply a machine config via `talosctl apply-config ...`.

So tl;dr is I think this should be workable. Ping me (or @frezbo, our resident SBC expert) in slack if you have issues.

**Reply by kfoxb** [reply]
Great! Thanks @rsmitty, I thought that might be the case, I’ll buy them and give it a go and let you know if I have any issues.

Thanks again!
